docker/Dockerfile.aarch64-linux-android

```diff
@@ -63,4 +69,7 @@ ENV CARGO_TARGET_AARCH64_LINUX_ANDROID_LINKER="$CROSS_TOOLCHAIN_PREFIX"gcc \
     TMPDIR=/tmp/ \
     ANDROID_DATA=/ \
     ANDROID_DNS_MODE=local \
-    ANDROID_ROOT=/system
+    ANDROID_ROOT=/system \
+    CROSS_CMAKE_SYSTEM_NAME=Android \
+    CROSS_CMAKE_SYSTEM_PROCESSOR=aarch64 \
```

Comment:
`CROSS_CMAKE_SYSTEM_PROCESSOR` is required for `CMAKE_SYSTEM_PROCESSOR` and `CROSS_CMAKE_SYSTEM_NAME` is just provided for consistency with every other toolchain.